src/sass/components/_hero-header.scss

```diff
@@ -0,0 +1,67 @@
+.container--header-bg {
+  background-color: $pinkBg;
+}
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-top: 6px;
+  &__hamburger {
+    background-color: transparent;
+    border: none;
+    cursor: pointer;
+  }
+}
+.container {
+  overflow: clip;
+  position: relative;
+}
+.hero {
+  background-color: $pinkBg;
+  margin-top: -1px;
```

Comment:
Jaki to ma cel? :)

Reply:
Wczoraj na operze miedzy hero a header przy zmianie rozdzielczości w niektórych momentach widać było biały pasek, dzisiaj nic takiego nie widzę więc nie wiem czy miałem jakiś błąd czy o co chodzi. W każdym razie ten 1px sprawiał że problem nie występował nigdy. Mogę to usunąć albo zostawić jako zabezpieczenie daj znać.